main.go

```diff
@@ -67,7 +67,7 @@ func (s *service) start() error {
 	listener, err := net.Listen("tcp", s.config.Listen)
 
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
It's still pointless. If server doesn't start main will finish, so Fatal does not change anything.

Real issue here is that all log messages are passed to syslog instead stdout. Best solution would be to add syslog/stdout switch in command line arguments an changes in config/config.go.